### PR TITLE
song ends after maximum utterances

### PR DESCRIPTION
**Pull request by Regexose**
@staudamm 
bitte in aller seelenruhe zerpflücken

### CODE REVIEW

File: processing/python/audience_view/surfaces.py [hunk 1]
```diff
@@ -45,12 +45,13 @@ class UtteranceLine:
     cat_surface = None
     
     def __init__(self, s_width, s_height, utt, cat, font, font_bold, pos_x):
+        print("utt {}  cat {}".format(utt,cat))
```
Comment (staudamm) [inline processing/python/audience_view/surfaces.py:48]: A print command in the constructor - always a bad idea

File: processing/python/audience_view/surfaces.py [hunk 2]
```diff
@@ -144,70 +145,70 @@ def calculate_circle_feed_positions(self):
...
+#     circle_centers = {}
+#     notify_info = {}
+#     i = 0
+#     a = 0
+#     start_x = 0
+#     start_y = 0
+#     goal_x = 0
+#     goal_y = 0
+#     circle_feed_positions = []
+#     col = color(0)
```
Comment (staudamm) [inline processing/python/audience_view/surfaces.py:158]: You might as well just delete the class, no?

File: song/song_server.py [hunk 3]
```diff
@@ -113,7 +132,8 @@ def _send_part(self, counter):
         self.performer_client.send_message(settings.SONG_BEAT_ADDRESS, message)
 
     def _send_utterance_to_audience(self, input_dict):
-        print(self.song_machine.category_counter, isinstance(self.song_machine.category_counter, dict))
+        # print(self.song_machine.category_counter, isinstance(self.song_machine.category_counter, dict))
+        print(input_dict)
```
Comment (staudamm) [inline song/song_server.py:136]: Can those print commands be removed?

**Review by staudamm — CHANGES_REQUESTED** [review]
@Regexose Solid work! Just a few minor comments on print commands

File: processing/python/audience_view/surfaces.py (hunk 2 above)
Comment (Regexose) [reply processing/python/audience_view/surfaces.py:158]: done

File: processing/python/audience_view/surfaces.py (hunk 1 above)
Comment (Regexose) [reply processing/python/audience_view/surfaces.py:48]: done

File: song/song_server.py (hunk 3 above)
Comment (Regexose) [reply song/song_server.py:136]: done

**Pull request merged**